hazelcast/src/main/java/com/hazelcast/aggregation/Aggregators.java

```diff
@@ -107,9 +107,9 @@ private Aggregators() {
      * @param <R> type of the return object.
      * @return an aggregator that calculates the distinct set of input values extracted from the given attributePath.
      * Accepts null input values and null extracted values.
-     * Aggregation result type is a Set of R.
+     * Aggregation result type is a CanonicalizingHashSet of R.
      */
-    public static <I, R> Aggregator<I, Set<R>> distinct(String attributePath) {
+    public static <I, R> Aggregator<I, CanonicalizingHashSet<R>> distinct(String attributePath) {
```

Comment:
`CanonicalizingHashSet` is an internal class. It should not be visible in the public API. I think this part should stay as `Set`

Reply:
Ok, changed.